CHANGELOG.md

```diff
@@ -14,7 +14,27 @@
 * one instance of mux in kafka plugin 
 * new field `group-id` can be added to kafka.conf. This value is used as a Group ID in order to set it 
   manually. In case the value is not provided, the service label is used instead (just like before). 
-      
+
+## Agent, Flavors
+Input arguments of `core.NewAgent()` has been changed (for backward compatibility the is `core.NewAgentDeprecated()`):
```

Comment:
Input arguments of the`core.NewAgent()` were changed (there is the `core.NewAgentDeprecated()` for backward compatibility):